server.js

```diff
@@ -2781,6 +2781,61 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CircleCI build integration.
+// https://circleci.com/api/v1/project/BrightFlair/PHP.Gt?circle-token=0a5143728784b263d9f0238b8d595522689b3af2&limit=1&filter=completed
+camp.route(/^\/circleci\/project\/([^\/]+\/[^\/]+)(?:\/(.*))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `doctrine/orm`.
```

Comment:
I didn't notice the comment! Because this is my first work in Node, I copied the travis implementation (which uses that comment). Thanks for pointing this out.